ladybug/datacollection.py

```diff
@@ -327,22 +356,23 @@ def update_data_for_analysis_period(self, values, analysis_period):
         """
         return self.update_data_for_hours_of_year(values, analysis_period.hoys)
 
-    def interpolate_data(self, timestep, half_hour=False):
-        """Interpolate data for a finer timestep.
+    def interpolate_data(self, timestep, cumulative=False):
+        """Interpolate data for a finer timestep using a linear interpolation.
```

Comment:
Would be great to add a test for this. That will also make it clear how the cumulative Boolean works.

Reply:
I might need some help on this one.  I'll give it a shot and post back here.

Reply:
Now that I am going through this, I think it might be better if I submit a test as a separate pull request after we merge this one in.  Given that this is the first test I'd be submitting, I would like you and @AntoineDao to get a clear look at it.

Reply:
Sure! let me know.

Reply:
Ok.  I committed a test.  It worked when I ran it locally.  Hopefully it makes sense.